core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -3825,7 +3825,8 @@ public boolean test(Project project) {
         + " sum(case when deptno = 10 then sal end) as sum_sal_d10,\n"
         + " sum(case when deptno = 20 then sal else 0 end) as sum_sal_d20,\n"
         + " sum(case when deptno = 30 then 1 else 0 end) as count_d30,\n"
-        + " count(case when deptno = 40 then 'x' end) as count_d40,\n"
+        + " sum(case when deptno = 40 then 1 end) as count_d40,\n"
```

Comment:
alias is `count` but the function is `sum`

Reply:
+1.

Reply:
> alias is `count` but the function is `sum`

See above `" sum(case when deptno = 30 then 1 else 0 end) as count_d30,\n"`, I followed this. "SUM CASE WHEN Then 1" means count, under my understanding.

Reply:
I see. I got confused because you edited an existing case (`count_d40` to `count_d10`) rather than adding new ones.

Reply:
The numbers are increasing sequence, edited an existing case to put them together, look more harmonious.